preupg/settings.py

```diff
@@ -43,6 +43,15 @@
 cache_dir = "/var/cache/preupgrade"
 log_dir = "/var/log/preupgrade"
 
+# file with meta info
+properties_ini = "properties.ini"
+# section inside ini file
+section_name = "preupgrade-assistant-modules"
+# key which specify source major OS version
+src_version_key ="srcMajorVersion"
+# key to specify destination major OS version
+dest_version_key ="destMajorVersion"
+
 # preupg log file
 preupg_log = os.path.join(log_dir, "preupg.log")
 
```

Comment:
One comment line per LOC is just way, way too many comments.  Please don't add comments unless you really have to; especially don't add comments that just say the same thing as the code says.

My advice would be to just remove all comments from this hunk. Or even better:

... because, wait ... why are there even "settings" for this? There's no need for this; these values are not really subject to change. IMO they should just be hard-coded into the hunk below, otherwise this just destroys readability (you have to look in *other file* just to see a "variable" that is never, ever supposed to change).